tst/teststandard/files/files.tst

```diff
@@ -0,0 +1,104 @@
+gap> START_TEST("files.tst");
+gap> testRead := function(testName, openFunc, lines, hasPosition)
```

Comment:
so... `testName` is only used for printing in error messages, presumably to identify the failing tests case? `openFunc` opens a new stream, `lines` contains the expected content of that stream, and `hasPosition` indicates whether `PositionStream` is supported. Correct?

Perhaps add this into a comment, and/or rename `lines` to `expectedLines` or `expected`; and `hasPosition` to `supportsPositionStream`?